src/components/web3/PermissionTypeSelector.tsx

```diff
@@ -31,17 +32,26 @@ const styles = {
     "&:hover": { background: "rgba(255, 255, 255, 0.42)" },
   }),
 };
+const disabledButton = css({
+  pointerEvents: "none",
+  opacity: 0.5,
+});
 
 export const PermissionTypeSelector: React.FC<Props> = ({
   setPermissionType,
   permissionType,
+  web3Account,
 }) => {
   const { t } = useTranslation();
   return (
     <div css={{ display: "flex" }}>
       <button
         onClick={() => setPermissionType("POAP")}
-        css={[styles.base, permissionType === "POAP" && styles.selected]}
+        css={[
+          styles.base,
+          permissionType === "POAP" && styles.selected,
+          web3Account === "SOL" ? disabledButton : null,
```

Comment:
nit: two different ways of optionally including styles right next to each other is a bit jarring.

I have no real preference between `condition && style` or `condition ? style : null` but please use the same syntax especially when the conditions are right next to each other. Otherwise the next reader of this code has to look twice to understand why these are differently phrased.